pages/project-version/middleware/index.js

```diff
@@ -1,4 +1,7 @@
 const { chain, get } = require('lodash');
+const { orderBy, remove, isEqual } = require('lodash');
+const moment = require('moment');
+const { traverse, getNode } = require('../../../lib/utils');
```

Comment:
these functions are specific to this implementation - let's not hide them in utils

Reply:
done